packages/icons/icons.json

```
@@ -1745,210 +1745,210 @@
         "codepoint": 62105
     },
     {
         "displayName": "Rotate: page",
         "iconName": "rotate-page",
-        "tags": "turn, anti clockwise",
+        "tags": "turn, anti clockwise, counterclockwise, left",
         "group": "editor",
         "codepoint": 62106
     },
     {
         "displayName": "Badge",
         "iconName": "badge",
-        "tags": "emblem, symbol, identification, insignia, marker",
+        "tags": "emblem, symbol, identification, insignia, marker, protect, privacy, shield, law enforcement, police, sheriff, rule enforcement, security",
         "group": "miscellaneous",
         "codepoint": 61743
     },
     {
         "displayName": "Grid view",
         "iconName": "grid-view",
-        "tags": "layout, arrangement",
+        "tags": "layout, arrangement, windows, boxes, squares, blocks, packing",
         "group": "editor",
         "codepoint": 61912
     },
     {
         "displayName": "Function",
         "iconName": "function",
-        "tags": "math, calculation",
+        "tags": "math, calculation, logic, fx, formula, spreadsheet",
         "group": "table",
         "codepoint": 61893
     },
     {
         "displayName": "Waterfall chart",
         "iconName": "waterfall-chart",
-        "tags": "graph, diagram",
+        "tags": "graph, diagram, bar chart, yield",
         "group": "data",
         "codepoint": 62230
     },
     {
         "displayName": "Stacked chart",
         "iconName": "stacked-chart",
-        "tags": "bar chart",
+        "tags": "bar chart, graph, statistics, summary",
         "group": "data",
         "codepoint": 62154
     },
     {
         "displayName": "Pulse",
         "iconName": "pulse",
-        "tags": "medical, life, heartbeat, hospital",
+        "tags": "medical, life, heartbeat, hospital, heart, graph, stock, trend, heart beat, alert, health",
         "group": "miscellaneous",
         "codepoint": 62084
     },
     {
         "displayName": "New person",
         "iconName": "new-person",
-        "tags": "person, human, male, female, character, customer, individual, add",
+        "tags": "person, human, male, female, character, customer, individual, add, user, additional",
         "group": "interface",
         "codepoint": 62039
     },
     {
         "displayName": "Exclude row",
         "iconName": "exclude-row",
-        "tags": "delete, remove, table",
+        "tags": "delete, remove, table, not equal, x, remove line, end point",
         "group": "table",
         "codepoint": 61854
     },
     {
         "displayName": "Pivot table",
         "iconName": "pivot-table",
-        "tags": "rotate, axis",
+        "tags": "rotate, axis, hinge, corner, lift, open, lever",
         "group": "table",
         "codepoint": 62070
     },
     {
         "displayName": "Segmented control",
         "iconName": "segmented-control",
-        "tags": "button, switch, option",
+        "tags": "button, switch, option, toggle, progress bar",
```

moved some of these from "array-boolean" to here, where they are more appropriate

```suggestion
        "tags": "button, switch, option, toggle, progress bar, boolean, erase, half",
```